src/TrustedShops.php

```diff
@@ -82,6 +82,27 @@ public static function woocommerce_thankyou_order_received_text($text, $order) {
   <span id="tsCheckoutOrderAmount">{$order->get_total()}</span>
   <span id="tsCheckoutOrderCurrency">{$order->get_currency()}</span>
   <span id="tsCheckoutOrderPaymentType">{$order->get_payment_method()}</span>
+EOD;
+    foreach ($order->get_items() as $item) {
+      $product = $item->get_product();
+      $product_id = $product->get_id();
+      $product_url = get_permalink($product_id);
+      $product_image = wp_get_attachment_url($product->get_data()['image_id']);
+      $product_sku = $product->get_sku();
+      $product_gtin = get_post_meta($product_id, '_custom_gtin');
+      $product_brand = !is_wp_error($brands = wp_get_post_terms($product_id, 'product_brand', ['orderby' => 'name', 'fields' => 'names'])) ? implode(' | ', $brands) : '';
```

Comment:
This is a little bit too compact, because https://developer.wordpress.org/reference/functions/wp_get_post_terms/#return states that there are not 2 but 3 possible return values:

> (array|WP_Error) Array of WP_Term objects on success or empty array if no terms were found. WP_Error object if $taxonomy doesn't exist.

```php
$brands = wp_get_post_terms($product_id, 'product_brand', ['orderby' => 'name', 'fields' => 'names']);
$product_brand = $brands && !is_wp_error($brands) ? implode(' | ', $brands) : '';
```

Reply:
#fyi @juanlopez4691